### DISCUSSION TITLE
Clipboard without screenshot copy/paste implemented？

### DISCUSSION THREAD

**Discussion by LusterDawn** [rustdesk/rustdesk#5725] [General]
### Bug Description

Though Text copy/paste works fine, the screenshot taken on local machine(via Shift+Win+S) cannot transfer to the remote, vice versa.
I used Wireshark to capture the network flow between the local and remote on LAN. I found there isn't any fluctuation after the screenshot was taken, not like the text.
I am a Rust beginner, for the moment I cannot tell whether there's implementation for Screenshot copy/paste in the code, so I made this Bug report.

### How to Reproduce

1. take screenshot via Shift+Win+S on local machine
2. check clipboard via Win+V on remote
3. no screenshot can be found on remote clipboard
vice versa

### Expected Behavior

The screenshot made on local machine(via Shift+Win+S) could transfer to the remote clipboard, just as text.

### Operating system(s) on local side and remote side

Windows10->Windows10

### RustDesk Version(s) on local side and remote side

1.2.2->1.2.2

### Screenshots

This is the network flow after I copied some plain text on local:
![image](https://github.com/rustdesk/rustdesk/assets/83108067/65b062b1-99b4-41ed-9f5b-29fdbb322396)

And this is the network flow after I made a screenshot on local:
![image](https://github.com/rustdesk/rustdesk/assets/83108067/819c25ce-7f63-4b60-929a-e177fed245fa)

**Comment by LusterDawn** [comment]
Found a temporary solution, use [ditto](https://ditto-cp.sourceforge.io/) as a clipboard instead of the one built in Rustdesk.
Ditto has a tab called Friends on its option panel, which enables full-feature clipboard share via network.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/683

**Comment by rustdesk** [comment]
ready now https://x.com/rustdesk/status/1807981430775206289
